packages/client/tests/functional/fulltext-search/__snapshots__/tests.ts.snap

```diff
@@ -25,3 +25,27 @@ Invalid \`.findMany()\` invocation in
 Error occurred during query execution:
 ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E42601), message: "syntax error in tsquery: \\"John Smith\\"", detail: None, hint: None, position: None, where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("tsquery.c"), line: Some(0), routine: Some("makepol") }) }), transient: false })
 `;
+
+exports[`fulltext-search (provider=postgresql, js_neon) bad query 1`] = `
+
+Invalid \`.findMany()\` invocation in
+/client/tests/functional/fulltext-search/tests.ts:0:0
+
+  XX // TODO: Edge: skipped because of the error snapshot
+  XX testIf(process.platform !== 'win32' && clientMeta.runtime !== 'edge')('bad query', async () => {
+  XX   const result = prisma.user
+→ XX     .findMany(
+syntax error in tsquery: "John Smith"
```

Comment:
Errors is identical to Postgres one, but there is is actually presented worse - wrapped in a `ConnectorError`. Interesting that the Driver Adapter leads to a better error here - but both should probably be some kind of known, properly handled error?

Reply:
I agree.